src/EPR.Accreditation.Facade/Controllers/OverseasSiteController.cs

```diff
@@ -40,5 +44,32 @@ public OverseasSiteController(IOverseasSiteService overseasSiteService)
 
             return Ok();
         }
+
+        [HttpGet("Outputs")]
+        [ProducesResponseType(typeof(OverseasReprocessingSiteOutputs), 200)]
+        public async Task<IActionResult> GetOverseasSiteOutputs(Guid accreditationExternalId, Guid siteExternalId)
+        {
+            var dto = await _accreditationService.GetOverseasReprocessingSiteOutputs(accreditationExternalId, siteExternalId);
+
+            if (dto == null)
+                return NotFound();
+
+            return Ok(dto);
+        }
+
+        [HttpPut("Outputs")]
+        public async Task<IActionResult> UpdateSite(
+            Guid accreditationExternalId,
+            [FromBody] OverseasReprocessingSiteOutputs overseasReprocessingSiteOutputs)
+        {
+            if (!overseasReprocessingSiteOutputs.ExternalId.HasValue)
```

Comment:
I'm looking at removing external id from the DTOs. I don't think it's needed as we can manage getting the parameters from the URLs. Can you double check and see if it can be done without this parameter? If so, can we also remove it from the class.

Reply:
This would require too many changes (DTO, BackEnd endpoint signature), so I've had to leave it for now. I can explain it further if you need.